Portfolio/src/Wrapper.vue

```diff
@@ -0,0 +1,34 @@
+<script setup lang="ts">
+</script>
+
+<template>
+    <div class="wrapper">
+
+        <slot></slot>
+    </div>
+</template>
+
+<style scoped>
+.wrapper {
+    font-family: "Raleway";
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin: 2rem;
+}
+
+@media (min-width: 280px) and (max-width: 767px) {
+    .wrapper {
+        align-items: center;
+        margin: 5rem;
+    }
+}
+
+@media (min-width: 1900px) {
+    .wrapper {
+        margin: 3rem 8rem 0 8rem;
+    }
+}
```

Comment:
I think u need to clean up these styles a bit. For example u start from adefault 2 rem margin, but is it for tablet? Between 767 and 1900? So I dont quite get what is going on here! 

What I would recommend is: Start with Mobile first, so u add a default margin of 5 rem, and then start scaling up for larger screens. Give me a call if u want to talk about it

Reply:
Done